pkg/services/alerting/notifiers/slack.go

```diff
@@ -353,7 +353,7 @@ func generateSlackBody(file string, token string, recipient string) (map[string]
 	var b bytes.Buffer
 	w := multipart.NewWriter(&b)
 	// Add the generated image file
-	f, err := os.Open(file)
+	f, err := os.Open(filepath.Clean(file))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.